kiliautoml/models/_hugging_face_named_entity_recognition_model.py

```diff
@@ -260,19 +262,18 @@ def _kili_assets_to_hf_ner_dataset(
         if not os.path.exists(path_dataset):
             with open(ensure_dir(path_dataset), "w") as handler:
                 for asset in tqdm(assets, desc="Converting assets to huggingface dataset"):
-                    self._write_asset(job_name, labels_to_ids, handler, asset)
+                    self._write_asset(job_name, labels_to_ids, handler, asset, label_merge)
 
         return label_list
 
-    def _write_asset(self, job_name, labels_to_ids, handler, asset):
+    def _write_asset(self, job_name, labels_to_ids, handler, asset, label_merge):
         text = self._get_text_from(asset["content"])
-        if (
-            job_name not in asset["labels"][0]["jsonResponse"]
-        ):  # always taking the first label (for now)
+        kili_label = get_label(asset, label_merge)
```

Comment:
I used kili_label because there is other variables named labels in some files that refers more or less to the json_response and not to the full kili label with author, type, etc. But it does not stop anything from working to just put label so I did it.